### DISCUSSION TITLE
How to archive `{admiral.test}`?

### DISCUSSION THREAD

**Discussion by manciniedoardo** [pharmaverse/admiral#1972] [General]
### Background Information

With the upcoming September release of  `{admiraldata}` we will be archiving a package (`{admiral.test}`) for the first time. How best to do this? Do we...
- Follow our deprecation guidance, having  `library(admiral.test)` throw a warning for a release cycle, pointing the user to use `admiraldata` instead and then archive the package later?
- Perform a "clean break" archiving `{admiral.test}` immediately and replacing all calls to it with calls to `{admiraldata}`?

@pharmaverse/admiral any thoughts?

### Definition of Done

Decision made on how to archive `admiral.test`

**Comment by bms63** [comment]
I think this is good opportunity to learn on how to deprecate a package and replace with another - and document what to do and not to do.  We kind of did this with `assertthat` when we created `admiraldev`, but that was an IMPORT package whereas this is a SUGGEST package.

**Comment by bundfussr** [comment]
I would follow our deprecation strategy. {admiral.test} is on CRAN and we do not know where it is used. It may be used in other places than our code. I think archiving the package without warning the users is not user-friendly.

**Comment by manciniedoardo** [comment]
# Outcome of discussion 2023-08-02 standup

- Publish pharmaversesdtm in sept release
- Add startup message in admiral.test at the same time directing users to pharmaversesdtm (sept release also)
- Roughly after 6 months to 1 year, contact CRAN and remove admiral.test entirely
